ci-operator/step-registry/health-check/node-count/health-check-node-count-commands.sh

```diff
@@ -17,18 +17,47 @@ then
     source "${SHARED_DIR}/proxy-conf.sh"
 fi
 
-control_plane_node_count=$(oc get node --no-headers | grep master | wc -l)
-compute_node_count=$(oc get node --no-headers | grep worker | wc -l)
+function all_nodes_ready()
```

Comment:
suggest to rename the function name to `all_nodes`, because the function does not check node status.

Reply:
Fixed